app/services/oid_minter_service.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+#
+class OidMinterService
+  def self.generate_oids(number)
+    oids = []
+    initialize_sequence!
+    number.times do
```

Comment:
In order to "minimize large gaps in sequence" - do we want to limit the number of oids a service is allowed to request at once? (scenario: a service accidentally requests 1,000 oids but only actually needs 10)

Reply:
I agree but wasn't sure what the max should be.  100 might be reasonable, our largest object is ~2,500 images.  This would be good for another ticket.